bigtable/google/cloud/bigtable/table.py

```diff
@@ -27,7 +29,29 @@
 from google.cloud.bigtable.row import ConditionalRow
 from google.cloud.bigtable.row import DirectRow
 from google.cloud.bigtable.row_data import PartialRowsData
-
+from google.gax import RetryOptions, BackoffSettings
+from google.cloud.bigtable.retry import ReadRowsIterator, _create_row_request
+from grpc import StatusCode
+
+BACKOFF_SETTINGS = BackoffSettings(
```

Comment:
Are these taken from the [GAPIC settings][]? If so, we should load from there; if not, we should alter those. They look to be pretty different; we should figure out what the correct values are.

  [GAPIC settings]: https://github.com/googleapis/api-client-staging/blob/master/generated/python/gapic-google-bigtable-v2/google/cloud/gapic/bigtable/v2/bigtable_client_config.json